test/e2e/tests/crtitical_tests_prs/test_community_permissions_add_edit_delete_duplicate.py

```diff
@@ -0,0 +1,152 @@
+from gui.main_window import MainWindow
+from gui.screens.community_settings import PermissionsIntroView
+from . import marks
+
+pytestmark = marks
+
+
+@allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/703632',
+                 'Manage community: Adding new permissions, Editing permissions, Deleting permission')
+@pytest.mark.case(703632, 705014, 705016)
+@pytest.mark.parametrize('params', [constants.community_params])
+@pytest.mark.critical
+def test_add_edit_remove_duplicate_permissions(main_screen: MainWindow, params):
+    with step('Enable creation of community option'):
+        settings = main_screen.left_panel.open_settings()
+        settings.left_panel.open_advanced_settings().enable_creation_of_communities()
+
+    main_screen.create_community(params['name'], params['description'],
+                                 params['intro'], params['outro'],
+                                 params['logo']['fp'], params['banner']['fp'])
+
+    permission_set = random.choice(permission_data)
+
+    with step('Open add new permission page'):
+        community_screen = main_screen.left_panel.select_community(params['name'])
+        community_setting = community_screen.left_panel.open_community_settings()
+        permissions_intro_view = community_setting.left_panel.open_permissions()
+
+    with step('Create new permission'):
+        permissions_settings = permissions_intro_view.add_new_permission()
+        permissions_settings.set_who_holds_checkbox_state(permission_set['checkbox_state'])
+        permissions_settings.set_who_holds_asset_and_amount(permission_set['first_asset'],
+                                                            permission_set['amount'])
+        permissions_settings.set_who_holds_asset_and_amount(permission_set['second_asset'],
+                                                            permission_set['amount'])
+        permissions_settings.set_is_allowed_to(permission_set['allowed_to'])
+        permissions_settings.set_in(permission_set['in_channel'])
+        permissions_settings.create_permission()
+
+    with step('Check toast message for permission creation'):
+        toast_messages = main_screen.wait_for_notification()
+        message = toast_messages[0]
+        assert ToastMessages.CREATE_PERMISSION_TOAST.value in toast_messages, \
+            f"Toast message is incorrect, current message is {message}"
+
+    with step('Created permission is displayed on permission page'):
+        if permission_set['asset_title'] is not False:
+            assert driver.waitFor(
+                lambda: permission_set['asset_title'] in permissions_settings.get_who_holds_tags_titles(),
+                configs.timeouts.UI_LOAD_TIMEOUT_MSEC)
+        if permission_set['second_asset_title'] is not False:
+            assert driver.waitFor(lambda: permission_set[
+                                              'second_asset_title'] in permissions_settings.get_who_holds_tags_titles(),
```

Comment:
weird line change here. Is it an auto formatting thing? I see it's done in a couple of places.

Reply:
ya, i think it has been done by IDE automatically to make the line length compliant with PEP8, looks weird indeed